bridge/opentracing/go.mod

```diff
@@ -7,19 +7,28 @@ replace go.opentelemetry.io/otel => ../..
 replace go.opentelemetry.io/otel/trace => ../../trace
 
 require (
+	github.com/opentracing-contrib/go-grpc v0.1.1
```

Comment:
go-grpc v0.1.2 requires go 1.23.8 or above

Reply:
We shouldn't upgrade then.